apps/app/src/components/UserSettingsDashboard/UserDispatcher.tsx

```diff
@@ -1,7 +1,75 @@
-import React from 'react'
+import { SetDispatcherRequest } from '@lens-protocol/client'
+import { signMessage } from '@wagmi/core'
+import React, { useState } from 'react'
+
+import lensClient from '@/lib/lens-protocol/lensClient'
+import { useAppPersistStore } from '@/store/app'
 
 const UserDispatcher: React.FC = () => {
-  return <p>Dispatcher</p>
+  const { currentUser } = useAppPersistStore()
+  const [isEnabled, setIsEnabled] = useState(currentUser!.dispatcher !== null)
```

Comment:
Changing `currentUser!.dispatcher` to `currentUser?.dispatcher` seems to fix the runtime error

Reply:
changed to currentUser? but now im getting errors for profileId and address: Type 'string | undefined' is not assignable to type 'string'.
  Type 'undefined' is not assignable to type 'string'.